<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>251</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0,0">
     <item row="0" column="1">
      <widget class="QPushButton" name="searchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Search</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="replaceCheckedBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace Checked</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QStackedWidget" name="stopAndNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QPushButton" name="nextButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopButton">
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="replaceCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="searchCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QWidget" name="searchOptionsLayoutW" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="matchCase">
          <property name="toolTip">
           <string>Match case</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="useRegExp">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable regular expressions. Find help for regex syntax by using the buttons in the search and replace text fields.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="expandResults">
          <property name="toolTip">
           <string>Expand results</string>
          </property>
          <property name="text">
           <string extracomment="When this is checked the result treeview will be expanded after a search"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="replaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="3">
      <widget class="QWidget" name="searchPlaceLayoutW" native="true">
       <layout class="QHBoxLayout" name="searchPlaceLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="searchPlaceCombo">
          <item>
           <property name="text">
            <string>In Current File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>In Open Files</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>In Folder</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="currentFolderButton">
          <property name="toolTip">
           <string>Use the current document's path.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QToolButton" name="displayOptions">
       <property name="toolTip">
        <string>Show more search options</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="newTabButton">
       <property name="toolTip">
        <string>Add new search tab</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="resultPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="resultTabWidget">
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="optionsPage">
      <layout class="QGridLayout" name="optionsLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="folderOptions" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="folderLabel">
            <property name="text">
             <string>Folder:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>folderRequester</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="KUrlComboRequester" name="folderRequester"/>
            </item>
            <item>
             <widget class="QToolButton" name="folderUpButton">
              <property name="toolTip">
               <string>Go one folder up.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="filterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>filterCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="10,0,10">
            <item>
             <widget class="KComboBox" name="filterCombo">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
              <item>
               <property name="text">
                <string>*</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="excludeLabel">
              <property name="text">
               <string>Exclude:</string>
              </property>
              <property name="buddy">
               <cstring>excludeCombo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="excludeCombo">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="recursiveCheckBox">
              <property name="text">
               <string>Recursive</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="hiddenCheckBox">
              <property name="text">
               <string>Include hidden</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="symLinkCheckBox">
              <property name="text">
               <string>Follow symbolic links</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="binaryCheckBox">
              <property name="text">
               <string>Include binary files</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchCombo</tabstop>
  <tabstop>replaceCombo</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceCheckedBtn</tabstop>
  <tabstop>searchPlaceCombo</tabstop>
  <tabstop>currentFolderButton</tabstop>
  <tabstop>matchCase</tabstop>
  <tabstop>useRegExp</tabstop>
  <tabstop>expandResults</tabstop>
  <tabstop>newTabButton</tabstop>
  <tabstop>displayOptions</tabstop>
  <tabstop>folderRequester</tabstop>
  <tabstop>folderUpButton</tabstop>
  <tabstop>filterCombo</tabstop>
  <tabstop>excludeCombo</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
  <tabstop>hiddenCheckBox</tabstop>
  <tabstop>symLinkCheckBox</tabstop>
  <tabstop>binaryCheckBox</tabstop>
  <tabstop>resultTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
